### ISSUE TITLE
Geo API doesn't work on EL8

### ISSUE THREAD

**Issue by bedroge** [EESSI/filesystem-layer#109]
@ocaisa  just noticed that his cvmfs client was complaining about the geo api.  It turns out that this doesn't work on Stratum 1 servers running EL8, as they don't have the `mod_wsgi` package for Python 2, and the cvmfs_geo api only works with Python 2. Support for Python 3 seems to be available on the `devel` branch already: https://github.com/cvmfs/cvmfs/pull/2829

This also shows that it doesn't work (it does with the `bgo-no` Stratum 1):
```
$ curl "http://rug-nl.stratum1.cvmfs.eessi-infra.org/cvmfs/pilot.eessi-hpc.org/api/v1.0/geo/DIRECT/aws-eu-west1.stratum1.cvmfs.eessi-infra.org,azure-us-east1.stratum1.cvmfs.eessi-infra.org,bgo-no.stratum1.cvmfs.eessi-infra.org,rug-nl.stratum1.cvmfs.eessi-infra.org"
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>500 Internal Server Error</title>
</head><body>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error or
misconfiguration and was unable to complete
your request.</p>
<p>Please contact the server administrator at 
 root@localhost to inform them of the time this error occurred,
 and the actions you performed just before this error.</p>
<p>More information about this error may be available
in the server error log.</p>
</body></html>
```
So, would be good to also add this to some test script/suite somewhere.

**Comment by boegel** [comment]
This should be fixed in CernVM-FS 2.9.1, see also https://sft.its.cern.ch/jira/browse/CVM-2052

**Comment by bedroge** [comment]
Fixed in 2.9.2 (see https://cvmfs.readthedocs.io/en/2.9/cpt-releasenotes.html), which I've installed on the affected Stratum 1 servers.
